pkg/sbom/scanner/scanner.go

```diff
@@ -77,6 +78,14 @@ func (s *Scanner) enoughDiskSpace(opts sbom.ScanOptions) error {
 	return nil
 }
 
+func sendResult(request *scanRequest, result *sbom.ScanResult) {
+	select {
+	case request.ch <- *result:
+	default:
+		log.Errorf("Failed to push scanner result for '%s' into channel", request.ID())
+	}
+}
```

Comment:
could you add a comment the attended behaviour of this function?

Reply:
Sure, however it was there in the beginning. I just moved it to make the code more readable.